docs/supported-languages.md

```diff
@@ -289,11 +283,20 @@ Their differences are outlined in the following table:
   </tr>
 </table>
 
-:::info Feature and product maturity levels
-* The detailed specifications previously provided apply only to language support. Language maturity levels differ from feature and product maturity levels.
-* Semgrep features and products documented as experimental, beta, or GA generally follow the definitions in a [Software release life cycle](https://en.wikipedia.org/wiki/Software_release_life_cycle).
-:::
+</details>
 
-#### Reachability support level
+### Feature and product maturity levels
+
+The detailed specifications previously provided apply only to language support. Language maturity levels differ from feature and product maturity levels.
+
+## More information
+Visit the cheat sheet generation script and associated semgrep-core test files to learn more about each feature:
+* [Generation script](https://github.com/semgrep/semgrep/blob/develop/scripts/generate_cheatsheet.py)
+* [`semgrep-core` test files](https://github.com/semgrep/semgrep/tree/develop/tests)
 
-Reachability support level refers to the level of support for reachability analysis for the language. At the minimum, Semgrep Supply Chain compares a package's version against a list of versions with known vulnerabilities
+Visit the Semgrep public language dashboard to see the parse rates for each language
+* See [Parse rates by language](https://dashboard.semgrep.dev/).
```

Comment:
Because we have a single bullet point here, can we just add the link to the line above?